bokeh/server/server.py

```diff
@@ -442,7 +442,7 @@ def __init__(self, applications: Mapping[str, Application | ModifyDoc] | Applica
 
             http_server = HTTPServer(tornado_app, **http_server_kwargs)
 
-            http_server.start(opts.num_procs)
+            http_server.install(opts.num_procs)
```

Comment:
Another one. This is a tornado `HTTPServer` not a Bokeh `Server` or `BaseServer`